app/components/ReproChecklist/ReproChecklist.js

```diff
@@ -3,36 +3,63 @@ import PropTypes from 'prop-types';
 import styles from './ReproChecklist.css';
 import ChecklistItem from './ChecklistItem/ChecklistItem';
 import Error from '../Error/Error';
-import { Typography, Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+import {
+  Typography,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material';
 import { SaveAlt } from '@mui/icons-material';
 import pdfMake from 'pdfmake/build/pdfmake';
 import pdfFonts from 'pdfmake/build/vfs_fonts';
 import GeneralUtil from '../../utils/general';
 import ChecklistUtil from '../../utils/checklist';
+import AssetUtil from '../../utils/asset';
 
 pdfMake.vfs = pdfFonts.pdfMake.vfs;
 
 const path = require('path');
 
+const scanFunctions = [
```

Comment:
I think it's okay to have it this way, but because this is coupling the order of the scan functions to the order of the checklist question order, you should add some comments here and where the order of the questions is set.  Just inform someone reading the code that if they change the order of the scan functions, they also need to modify <other file location>, and vice versa.

Reply:
Ah yes! Good point!